### DISCUSSION TITLE
Custom Layout For Login Component

### DISCUSSION THREAD

**Discussion by Sharafudeen** [oqtane/oqtane.framework#1128] [Q&A]
Hi,

How do I do custom layout page for login component. Currently login component is rendering under Pages/_Host.cshtml in Oqtane.Server project. Can we have more than one layout page for specific components?

Thanks!

**Comment by sbwalker** [comment]
@Sharafudeen Oqtane is built on Blazor and utilizes dynamic razor components to construct the user interface. In Oqtane "pages" are a logical concept - not a physical code file... you can define as many "pages" as you want within a "site" grouping however they are simply metadata stored in the database. Each page can have a custom layout - in Oqtane layouts are part of a "theme" that are simply a collection of razor component that includes "panes" where module content can be dynamically injected at runtime. The core framework contains 2 themes in the Oqtane.Client\Themes folder - the BlazorTheme and OqtaneTheme. And if you want to create an external custom theme you can look at https://github.com/oqtane/oqtane.theme.cerulean as an example.

**Reply by Sharafudeen** [reply]
Login page also contains menus and footer like other pages. If I wanted to remove menus and footer for login page then what should I do?

Already I've gone through suggested link for themes but it has menus in the login page.

For the cerulean theme, how we can remove menus in login page?

Thanks!

**Reply by erw13n** [reply]
Login is just another page, you can modify the Page Setting and set a different Theme that doesn't have menu and footer.
